scripts/release/version_doc.py

```diff
@@ -0,0 +1,34 @@
+import yaml
+
+
+def table(list_of_dicts):
```

Comment:
this seems pretty fragile.  It's assuming:
1. The compatibility YAML has at least 1 dict item
2. All the dict items have exactly identical keysets

Reply:
> The compatibility YAML has at least 1 dict item
Added a condition for this 

> All the dict items have exactly identical keysets
it's a table so that's what we're expecting, I just put in a pre-condition for this,  I'd rather avoid spending a lot of time for rigorous validation here as it's just a utility script.